### PR TITLE
changes Loading room... to Loading...

### PR DESCRIPTION
**Pull request by abhijain2003**
I have changed the Loading room... to Loading...
in GroupCallView and GroupCallLoader files to maintain consistency in loading component in web.

targeting issue 
https://github.com/vector-im/element-call/issues/461#issuecomment-1422197846
#461

### CODE REVIEW

**Review by SimonBrandner — CHANGES_REQUESTED** [review]
Thank you though you need to run the `yarn i18n` script now

**Comment by abhijain2003** [comment]
I did I think

**Comment by SimonBrandner** [comment]
Did you commit the changes the script did?

**Comment by abhijain2003** [comment]
should I do it again

**Comment by SimonBrandner** [comment]
Can you try again? It seems the script did no changes on your side, while on my side it does change `app.json` with your changes

**Comment by abhijain2003** [comment]
as you said, I just did it again. let me know now it is working or not

**Comment by SimonBrandner** [comment]
Can you write down everything you did?

**Comment by abhijain2003** [comment]
I just run a command yarn i18n in the root directory.

**Comment by abhijain2003** [comment]
do you want me write what and where I have made changes as well?

**Comment by SimonBrandner** [comment]
> do you want me write what and where I have made changes as well?

Yep, I still can't see what I should see... it would help me figure out what's going on

**Comment by abhijain2003** [comment]
Screenshot 2023-02-09 133232


the only thing I did was

**Comment by SimonBrandner** [comment]
I meant what you did in the terminal, what commands you ran, what files changed, how you committed the changes etc.

**Comment by abhijain2003** [comment]
sorry for that, 

ok, so commands I run (after changes).
yarn i18n
git init
get remote add upstream <original repo link>
get fetch upstream 
git checkout -b abhijain2003/loginui
git add .
git commit -m "message"
git push origin my_branch

**Comment by SimonBrandner** [comment]
No worries, can you try this and paste the terminal output:

```
git checkout abhijain2003/loadingui
git pull
yarn i18n
git add .
git commit - m "i18n"
```

**Comment by abhijain2003** [comment]
sure

**Comment by SimonBrandner** [comment]
And also `git push` in the end :)

**Comment by abhijain2003** [comment]
when I am doing all these, there is no changes is happening, in code
Screenshot 2023-02-09 141016

as you can see there is nothing to commit is coming

**Comment by abhijain2003** [comment]
do you have any update for this, is there any way I can help you

File: src/room/GroupCallLoader.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ export function GroupCallLoader({
   if (loading) {
     return (
       <FullScreenView>
-        <h1>{t("Loading room…")}</h1>
+        <h1>{t("Loading...")}</h1>
```
Comment (robintown) [inline src/room/GroupCallLoader.tsx:54]: Please use the `…` Unicode character rather than 3 dots, for consistency with our other strings

**Review by robintown — CHANGES_REQUESTED** [review]

**Comment by abhijain2003** [comment]
sure

**Comment by abhijain2003** [comment]
I have commited the requested changes,

**Comment by SimonBrandner** [comment]
Hey, sorry for the delay here! This would be cool to merge but it's not clear to me what is going on here with the `i18n` script. Do you think you'll be able to figure it out or should I take over?

**Comment by abhijain2003** [comment]
I think I can, but these days I am busy with my freelance clients I would need some time, but for the community i think the best is you should take it over to someone else

highly apologies for this.

will surely contribute to the community in different way

**Comment by SimonBrandner** [comment]
> highly apologies for this.

No worries at all :)

**Pull request merged**